programs/native/bpf_loader/src/lib.rs

```diff
@@ -161,7 +161,7 @@ fn entrypoint(
 
     if keyed_accounts[0].account.executable {
         let prog = keyed_accounts[0].account.userdata.clone();
-        trace!("Call BPF, {} instructions", prog.len() / 8);
+        info!("Call BPF program");
```

Comment:
Can I meet you in the middle with `debug!`?  I'm generally trying to keep `info!` from getting too spammy

Reply:
I think these two messages should be on the same level as:
`INFO  solana_bpf_loader] Finalize: account BMjCmE4mWPm3VCJGm8m2h9hLp2ZAWbhBcagc8gmtJixw`
But at a higher level then:
`DEBUG solana_bpf_loader] Write: offset=0 length=256`

If we move the two modified messages in this pr to `debug` I think we should move the `Write` messages from `debug` to `trace`.

Reply:
@mvines, what do you think about my last comment?

Reply:
Let's keep them as info for now.  They are helpful for development I guess.  I'm worried about log spam with a high transaction rate, but that probably should be solved at a more system level than locally here.  :shipit: